runtime/devnet/src/config/ismp.rs

```diff
@@ -17,7 +17,7 @@ impl pallet_ismp::Config for Runtime {
 	type Coprocessor = Coprocessor;
 	type Currency = Balances;
 	type HostStateMachine = HostStateMachine;
-	type Mmr = pallet_ismp::NoOpMmrTree<Self>;
+	type OffchainDB = ();
```

Comment:
Confirmed with polytope labs team that this is oke to use in stead of the [Mmr pallet](https://github.com/polytope-labs/hyperbridge/blob/a2cb646d3aee28e04755c704bba1cf7a3672b333/parachain/runtimes/nexus/src/ismp.rs#L113)